backend/_handler.js

```diff
@@ -4,106 +4,110 @@ import * as db from '@reshuffle/db';
 const swaggerUi = require('swagger-ui-express');
 const swaggerDocument = require('./swagger.json');
 
-
 const app = express();
 
 // API endpoints
 const allPetsQuery = db.Q.filter(db.Q.key.startsWith('pets/'));
 
 //GET /collection - gets all entries in the collection, query string is the filter attributes like /collection?param1=foo
 app.get('/api/pets', async (req, res) => {
-    try {
-        const result = await db.find(allPetsQuery);
-        res.setHeader('Content-Type', 'application/json');
-        res.end(JSON.stringify(result));
-    } catch (e) {
-        res.sendStatus(500);
-        console.error(e);
-    }
+  try {
+    const result = await db.find(allPetsQuery);
+    res.setHeader('Content-Type', 'application/json');
+    res.end(JSON.stringify(result));
+  } catch (e) {
+    res.sendStatus(500);
+    console.error(e);
+  }
 });
 
-// POST /collection - create an new entry. The ID will return as the “id” field in the Json response 
+// POST /collection - create an new entry. The ID will return as the “id” field in the Json response
 app.post('/api/pets', express.json(), async (req, res) => {
   try {
-      const value = req.body;
-      var id = value.id;
-      if(!id || id == 0){
-          id = uuidv4();
-          value.id  = id;
-      }
-      var response =  await db.update(`pets/${id}`, (prev_value) => { return value; });
-      res.setHeader('Content-Type', 'application/json');
-      res.end(JSON.stringify(response));
+    const value = req.body;
+    var id = value.id;
+    if (!id || id == 0) {
+      id = generateUid();
+      value.id = id;
+    }
+    var response = await db.update(`pets/${id}`, (prev_value) => {
+      return value;
+    });
+    res.setHeader('Content-Type', 'application/json');
+    res.end(JSON.stringify(response));
   } catch (e) {
-      res.sendStatus(500);
-      console.error(e);
+    res.sendStatus(500);
+    console.error(e);
   }
 });
 
 //GET /collection/:id - gets just one entry. This is the proper way to get a single ID and not using a filter command.
 app.get('/api/pets/:id', express.json(), async (req, res) => {
   try {
-      const id = req.params.id;
-      var key = `pets/${id}`;
-      const result = await db.get(key);
-      
-      res.setHeader('Content-Type', 'application/json');
-      res.end(JSON.stringify(result));
+    const id = req.params.id;
+    var key = `pets/${id}`;
+    const result = await db.get(key);
+
+    res.setHeader('Content-Type', 'application/json');
+    res.end(JSON.stringify(result));
   } catch (e) {
-      res.sendStatus(500);
-      console.error(e);
+    res.sendStatus(500);
+    console.error(e);
   }
 });
 
 //PUT /collection/:id - updates an entry. Should return the updated entry.
```

Comment:
The documentation says return the updated entry, but the `oldEntry` is returned.